packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -735,13 +764,22 @@ export const Webchat = forwardRef((props, ref) => {
               <ErrorMessage>{webchatState.error.message}</ErrorMessage>
             </ErrorMessageContainer>
           )}
-
-          <WebchatMessageList />
-
-          {webchatState.replies &&
-            Object.keys(webchatState.replies).length > 0 && (
-              <WebchatReplies replies={webchatState.replies} />
-            )}
+          <div
+            id={BotonicContainerId.ChatArea}
+            style={{
+              display: 'inherit',
+              flexDirection: 'inherit',
+              height: chatAreaHeight,
+              width: 'inherit',
+              overflow: 'inherit',
+            }}
```

Comment:
These styles should be in a styled component. 
I think there is no need to add a div here. We could do this inside the WebchatMessageList component.

Reply:
done in the next PR